### PR TITLE
Extract methods

### PR DESCRIPTION
**Pull request by lilia-project**

### CODE REVIEW

File: src/BaseConverter/BaseConverter.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package BaseConverter;
+
+import java.util.Scanner;
+
+public class BaseConverter {
+    public static void main(String[] args) {
+        do {
+            Scanner scanner = new Scanner(System.in);
+            methodConvert();
```
Comment (edvardino) [inline src/BaseConverter/BaseConverter.java:9]: Good chunk for extraction!  Try to pick up appropriate name for the method. It is not clear what it does at the moment

File: src/BaseConverter/BaseConverter.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            int option = scanner.nextInt();
+
+            if (inputValidation(option)) return;
```
Comment (edvardino) [inline src/BaseConverter/BaseConverter.java]: The same, it is a good part of code to extract to the separate method. But I would recommend using method name like isInvalidChoice(option) or invalidSelection(option). It becomes much simpler for understanding. You start reading a code like a book: if option - invalid selection then we do return. Accordingly, to conventions method name should be a verb but in the case of `if` construction and predicate (it is a method that returns a boolean value)

**Review by edvardino — CHANGES_REQUESTED** [review]
Nice transformations! Read comments and try to do simple refactoring 👍

File: src/BaseConverter/BaseConverter.java (hunk 2 above)
Comment (lilia-project) [reply src/BaseConverter/BaseConverter.java]: I fixed. 
Upload to the same branch?

Comment (edvardino) [reply src/BaseConverter/BaseConverter.java]: correct

Comment (lilia-project) [reply src/BaseConverter/BaseConverter.java]: I forgot files

Comment (edvardino) [reply src/BaseConverter/BaseConverter.java]: So, where are the changes?

**Review by edvardino — APPROVED** [review]
Do merge by yourself 💯

**Pull request merged**